### PR TITLE
Build long prefix package using conda-build 2.x

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

**Comment by ocefpaf** [comment]
Jonathan thanks for doing this! I am +1 to this approach here and the other feedstock that needs the long prefix.

**Comment by jakirkham** [comment]
Does `conda inspect prefix-lengths` work in `test/commands`? If it does, it may be worth keeping that around. Just a thought.

**Comment by jjhelmus** [comment]
> Does conda inspect prefix-lengths work in test/commands?

It may, but I'm not sure how to best determine the path and name of the package being tested.  Hard coding these using the conda-forge directories seems like a bad idea as the recipes would not longer be general purpose.

File: .travis.yml [hunk 1]
```diff
@@ -20,22 +23,26 @@ before_install:
...
+      source run_conda_forge_build_setup
+
+      # install conda-build 2.x to build with a long prefix
+      conda install --yes --quiet conda-build=2
+      conda info
 
 script:
-  - conda build ./recipe
+  # Use the default channel as a provider of long prefix packages
+  - conda build -c defaults ./recipe
```
Comment (ocefpaf) [inline .travis.yml:43]: Clever way to workaround the non-conda-build-2 dependencies :+1:

**Pull request merged**